assets/styles/informed.scss

```diff
@@ -2090,3 +2090,8 @@ caption {
   background-color: #f3f2f1;
   padding: 1rem 1rem 0 1rem;
 }
+
+.govuk-notification-banner--error {
+  background-color: #d4351c;
+  border-color: #d4351c;
+}
```

Comment:
Im surprised there isn't a set of colour variables (colour theme). Not suggesting you create them now.